image/templates/helm/stackrox-secured-cluster/templates/collector.yaml

```diff
@@ -143,6 +143,7 @@ spec:
         - mountPath: /run/secrets/stackrox.io/certs/
           name: certs
           readOnly: true
+      {{- if eq ._rox.env.openshift 4 }}
```

Comment:
We have to remember about this when OpenShift 5 comes 🙂 

Jokes aside, it would be good to have a more reliable env. Like `cluster_flavor` or something like that, which is not tied to the version. If that doesn't exist I'm fine with leaving this as is.

Reply:
`{{- if eq ._rox.env.openshift 4 }}` seems to be pretty abundant in our helm templates. If OS 5 comes out, we well have some work to do ;)
I will allow myself to leave it as it is for now